stripe-servant/src/Stripe/Resources.hs

```diff
@@ -285,7 +367,19 @@ instance ToForm CustomerCreate where
   toForm = genericToForm (formOptions 2)
 
 instance ToForm CustomerUpdate where
-  toForm = genericToForm (formOptions 2)
+  toForm cu =
```

Comment:
ooc: does the automatic deriving not work here? my assumption it only doesn't work for nested hashes in arrays. Either way, we can also leave it as is :)